climada/hazard/trop_cyclone.py

```diff
@@ -792,35 +776,35 @@ def _compute_windfields_sparse(
             max_memory_gb=max_memory_gb,
         )
 
-    windfields, reachable_centr_idx = _compute_windfields(
-        si_track, coastal_centr, mod_id, metric=metric, max_dist_eye_km=max_dist_eye_km,
+    windfields, idx_centr_reachable = _compute_windfields(
+        si_track, centroids_close, mod_id, metric=metric, max_dist_eye_km=max_dist_eye_km,
     )
-    reachable_coastal_centr_idx = coastal_idx[reachable_centr_idx]
+    idx_centr_filter = idx_centr_filter[idx_centr_reachable]
     npositions = windfields.shape[0]
 
     intensity = np.linalg.norm(windfields, axis=-1).max(axis=0)
     intensity[intensity < intensity_thres] = 0
     intensity_sparse = sparse.csr_matrix(
-        (intensity, reachable_coastal_centr_idx, [0, intensity.size]),
+        (intensity, idx_centr_filter, [0, intensity.size]),
         shape=intensity_shape)
     intensity_sparse.eliminate_zeros()
 
     windfields_sparse = None
     if store_windfields:
-        n_reachable_coastal_centr = reachable_coastal_centr_idx.size
-        indices = np.zeros((npositions, n_reachable_coastal_centr, 2), dtype=np.int64)
-        indices[:, :, 0] = 2 * reachable_coastal_centr_idx[None]
-        indices[:, :, 1] = 2 * reachable_coastal_centr_idx[None] + 1
+        n_centr_filter = idx_centr_filter.size
+        indices = np.zeros((npositions, n_centr_filter, 2), dtype=np.int64)
+        indices[:, :, 0] = 2 * idx_centr_filter[None]
+        indices[:, :, 1] = 2 * idx_centr_filter[None] + 1
         indices = indices.ravel()
-        indptr = np.arange(npositions + 1) * n_reachable_coastal_centr * 2
+        indptr = np.arange(npositions + 1) * n_centr_filter * 2
         windfields_sparse = sparse.csr_matrix((windfields.ravel(), indices, indptr),
                                               shape=windfields_shape)
         windfields_sparse.eliminate_zeros()
 
     return intensity_sparse, windfields_sparse
 
 def _compute_windfields_sparse_chunked(
-    track_centr_msk: np.ndarray,
+    mask_close_alongtrack: np.ndarray,
```

Comment:
I know that this renaming is not clarifying everything perfectly. But at least the naming is more consistent within `trop_cyclone.py` regarding the aspect of using the `mask_` prefix (instead of a `_msk` suffix) to indicate that this is a bool array.

Reply:
Maybe make clear that this is a mask for the centroids?
```suggestion
    mask_centr_along_track: np.ndarray,
```
Also, why does this take the centroids and a mask separately, and not just the masked centroids?

Reply:
The `_alongtrack` part means that it's not just a single mask, but it's an "along-track mask", meaning that it is actually one mask for each track position. That's why the array is not one-dimensional, but two-dimensional. This is already made clear from the docstrings:
https://github.com/CLIMADA-project/climada_python/blob/83cff963a9d1f2311cfd16122a9e0b0dd54334d3/climada/hazard/trop_cyclone.py#L817-L818